server/cshr/routs/training_courses.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from ..views.training_courses import TrainingCoursesAPIView
+
+urlpatterns = [
+    path(
+        "<str:id>/",
+        TrainingCoursesAPIView.as_view(
+            {"get": "get_one", "delete": "delete", "put": "put"}
+        ),
+    ),
+    path("all/", TrainingCoursesAPIView.as_view({"get": "get_all"})),
+    path("", TrainingCoursesAPIView.as_view({"post": "post"})),
```

Comment:
Why here do we have 2 routes? it can be done with just one.
```suggestion
    path("", TrainingCoursesAPIView.as_view({"get": "get_all", "post": "post"})),
```